### PR TITLE
[@type/conflux-web fixed the bugs]

### PR DESCRIPTION
**Pull request by liuis**
Please fill in this template.

- [ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/conflux-web/tslint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
```
Comment (sheetalkamat) [inline types/conflux-web/tslint.json]: Remove these rules from here and fix error. New packages cannot disable any rules or use custom rules.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/conflux-web/conflux-web-tests.ts [hunk 2]
```diff
@@ -0,0 +1,184 @@
+import Web3 = require("conflux-web");
```
[the remark this answers is not in the document]
Comment (liuis) [reply types/conflux-web/conflux-web-tests.ts]: Not web3, is indeed a conflux-web, our folk web3 project to support our project.

[the remark this answers is not in the document]
Comment (sheetalkamat) [reply types/conflux-web/conflux-web-tests.ts]: Please fix the project link in the index.d.ts then

File: types/conflux-web/tslint.json (hunk 1 above)
Comment (liuis) [reply types/conflux-web/tslint.json]: can't remove, when I remove , error! 

_"ERROR: 12:6   strict-export-declare-modifiers  All declarations in this module are exported automatically. Prefer to explicitly write 'export' for clarity. If you have a good reason not to export this declaration, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md"_

File: types/conflux-web/conflux-web-tests.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (liuis) [reply types/conflux-web/conflux-web-tests.ts]: thx， fixed

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
This package does contain its own types so these definitions will not be accepted here. Please refer to checkbox in the PR template containing The package does not already provide its own types

**Comment by liuis** [comment]
> This package does contain its own types so these definitions will not be accepted here. Please refer to checkbox in the PR template containing The package does not already provide its own types

Our project is, indeed, the fork the web3 projects, but there are a lot of difference, but in the types of content is the same.So the web3 type is our.

**Pull request closed without merging**